engine/src/main/java/org/terasology/engine/core/modes/loadProcesses/InitialiseWorld.java

```diff
@@ -93,7 +93,7 @@ public boolean step() {
             worldInfo.setSeed(random.nextString(16));
         }
 
-        logger.info("World seed: \"{}\"", worldInfo.getSeed());
+        logger.atInfo().addArgument(() -> worldInfo.getSeed()).log("World seed: \"{}\"");
```

Comment:
Complexity and impact of `getSeed()` are neglectable, ignore using comment:
```suggestion
        logger.info("World seed: \"{}\"", worldInfo.getSeed()); //NOPMD
```